runtime/testy/Ansible.py

```diff
@@ -223,15 +230,99 @@ def start(self):
         try:
             while True:
                 nextCall = time.time()
+                nextCall += 1.0/UDPRecvClass.UDP_RECV_HZ
```

Comment:
You have a call to `select` right under this, which should be blocking. In that case, there should be absolutely no reason to have sleeps anywhere in this loop.